src/utils/eis_utils/src/eis_err.c

```diff
@@ -58,6 +58,8 @@ const char* EISErr_ErrToString(EISErr eisErr)
         return "EISErr_ContentAllocErr";
     case EISErr_InvalidJsonRespErr:
         return "EISErr_InvalidJsonRespErr";
+    case EISErr_NamedPipeFailure:
```

Comment:
Consider more descriptive name here. e.g., EISErr_RespNamedPipeInitErr, or something along this line. In case we want to add read or write error codes in the future.